test/test_other.py

```diff
@@ -3359,7 +3359,12 @@ def test_jspi_add_function(self):
       '-Wno-experimental']
     self.do_runf('other/test_jspi_add_function.c', 'done')
 
-  def test_embind_tsgen(self):
+  @parameterized({
+    '': [[]],
+    'with_jsgen': [['-sEMBIND_AOT']]
+  })
+  def test_embind_tsgen(self, opts):
+    self.emcc_args += opts
```

Comment:
How about `emcc_args=['-lembind', '--emit-tsd', 'embind_tsgen.d.ts'] ) opts` below instead of add a new line here?